src/db/db.js

```diff
@@ -23,12 +23,16 @@ class Database {
   static async createScripts() {
     const con = Database.connection();
     await con.query(`
+    DROP TABLE ADMIN IF EXISTS;
```

Comment:
we already have scripts for creating and dropping the tables,

Reply:
Those scripts were not well written, <IF EXISTS> should come after main query instead of beginning